lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -417,8 +417,8 @@ sub run_backup {
         die $request->{_locale}->text('Error creating backup file');
 
     if ($request->{backup_type} eq 'email') {
-        # suppress warning of single usage of $LedgerSMB::Sysconfig::...
-        no warnings 'once';
```

Comment:
If manual testing is too onerous, I would be ok with accepting this line and filing a bug on the warning.  I am guessing that the problem is that $csettings does not appear to be used following.  And that should be a fairly easy one to isolate and fix.